docs/Buttons/OnButtonPress/index.md

```diff
@@ -9,7 +9,7 @@ Sender
 Purpose
 : Inform SDL about a Button Press
 
-If the HMI reports to SDL via [Buttons.GetCapabilities](../getcapabilities) that it supports long and/or short button press modes, SDL expects the HMI to send the Buttons.OnButtonPress notification but buttons that have been subscribed via [Buttons.OnButtonSubscription](../onbuttonsubscription) and custom buttons added in other RPCs as Soft Buttons.
+If the HMI reports to SDL via [Buttons.GetCapabilities](../getcapabilities) that it supports long and/or short button press modes, SDL expects the HMI to send the Buttons.OnButtonPress notification but buttons that have been subscribed via [Buttons.OnButtonSubscription](../../buttons/subscribebutton/) and custom buttons added in other RPCs as Soft Buttons.
```

Comment:
```suggestion
If the HMI reports to SDL via [Buttons.GetCapabilities](../getcapabilities) that it supports long and/or short button press modes, SDL expects the HMI to send the Buttons.OnButtonPress notification but buttons that have been subscribed via [Buttons.SubscribeButton](../../buttons/subscribebutton/) and custom buttons added in other RPCs as Soft Buttons.
```

Since `OnButtonSubscription` was removed and replaced by `SubscribeButton` I think we should change the link title as well